src/bin/pgcopydb/pgsql.c

```diff
@@ -1913,9 +1913,8 @@ pgsql_enable_pipeline_mode(PGSQL *pgsql)
 	if (!warned)
 	{
 		log_warn("Skipping libpq pipeline mode optimisation because pgcopydb "
-				 "was built with libpq %d, pipeline mode is available since "
-				 "libpq 14",
-				 PG_MAJORVERSION_NUM);
+				 "was built with libpq " PG_MAJORVERSION
```

Comment:
I'm wondering what is the purpose of this change? PG_MAJORVERSION_NUM is an integer and we were using %d format specifier to print, I don't see any flaw with that.

Reply:
support postgres 11 and 12

Reply:
`PG_MAJORVERSION_NUM` have been used in other places too. I'm not sure why the problem is not pronounced there?

https://github.com/search?q=repo%3Adimitri%2Fpgcopydb%20PG_MAJORVERSION_NUM&type=code

It would be great if you can write a description of why you need this change along with the problem.

Thank you.

Reply:
Ok, after greping into PG source, I see that `PG_MAJORVERSION_NUM` doesn't exists on PG11 and it has only `PG_MAJORVERSION`. Please update PR title and commit message to reflect the intent of this change.

Reply:
> `PG_MAJORVERSION_NUM` have been used in other places too. I'm not sure why the problem is not pronounced there?

I don't understand, why problem is not appeared in other places, but I prefer change them to universal check like
```c
#if PG_VERSION_NUM >= 150000
```